src/Raven.Studio/typescript/common/shell/accessManager.ts

```diff
@@ -1,71 +1,205 @@
 ﻿/// <reference path="../../../typings/tsd.d.ts"/>
+import activeDatabaseTracker = require("common/shell/activeDatabaseTracker");
 
 class accessManager {
 
     static default = new accessManager();
     
     static clientCertificateThumbprint = ko.observable<string>();
     
+    static databasesAccess: dictionary<Raven.Client.ServerWide.Operations.Certificates.DatabaseAccess> = {};
+    
     securityClearance = ko.observable<Raven.Client.ServerWide.Operations.Certificates.SecurityClearance>();
 
     private allLevels = ko.pureComputed(() => true);
     
     // cluster node has the same privileges as cluster admin
-    clusterAdminOrClusterNode = ko.pureComputed(() => this.securityClearance() === "ClusterAdmin" || this.securityClearance() === "ClusterNode");
-        
-    operatorAndAbove = ko.pureComputed(() => { 
+    isClusterAdminOrClusterNodeClearance = ko.pureComputed(() => {
```

Comment:
do we need to add word Clearance in those variables? (in other words we know variable type - no need to induce type in variable name).

Reply:
done